### ISSUE TITLE
Multi user chat : is that possible ?

### ISSUE THREAD

**Issue by yarekc** [rtc-io/rtc#24]
Does it relay streams ?
My goal is to create some video chat application with 20 users: and webrtc seems to have limits to 6 beacuse of its peer2peer technology.

**Comment by silviapfeiffer** [comment]
Hi @yarekc there is no actual limit to endpoints - the limit of 6 is artificial and more based on the resources that the endpoints can deal with. That's particularly problematic with video because of the large amounts of data, but less so with text chat.

If you want a video chat application with many endpoints, you need to install a media server (MCU or  SFU).

**Comment by yarekc** [comment]
My idea was rather to use a chrome or chromium on server side as relay without using complex sfu servers
Do you think my idea can work ?
Regards

**Comment by silviapfeiffer** [comment]
Chromium is not a forwarding server. All you get by running it in the cloud is another endpoint. You'd have to implement SFU functionality in JS. If that's your goal, sure. Rtc.io is not helping with that. [sorry about the earlier autocomplete typos]

**Comment by yarekc** [comment]
yes I know it is not a SFU server. However, it looks like it is possible nowdays to relay peers. So de factor, it can play a SFU role, well I believe it. 
I have tried many SFU servers and they all are complex and have big CPU consumption. Whereas we just need to relay them without any process.  I still believe there must be a simple solution.

**Comment by silviapfeiffer** [comment]
Hmm, maybe try https://github.com/versatica/mediasoup ?

**Comment by yarekc** [comment]
Yes: I already did as well as Janus or Kurento... and I realized that maybe any browser can do what these servers do: simply relay a peer : 
so Instead of having: 
performer<----->user1
performer<----->user2

(that will kill performer if we have 100 users)

why not have
performer<----->chrome server <---->user1   
___________________________________<----->user2

so remote chrome acts as simply relay between performer and users

(sorry for the underline but these comments do not accept SPACE or TABs)
